src/pages/people-page/PeoplePage.tsx

```diff
@@ -0,0 +1,118 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { useParams } from 'react-router-dom';
+
+export const PeoplePage = () => {
+  const [peopleList, setPeopleList] = useState<Person[]>([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const { peopleSlug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(res => {
+        const peopleRes: Person[] = [];
+
+        res.map(person => {
+          const mother = res.find(p => p.name === person.motherName);
+          const father = res.find(p => p.name === person.fatherName);
+
+          peopleRes.push({ ...person, mother: mother, father: father });
```

Comment:
When declaring an object like this, based on other consts, if the obejct's property has the same name as the const, you can pass only the const, without the property declaration. 
What I mean is that the following code line does the same as your code:
```
peopleRes.push({ ...person, mother, father });
```
Notice how "father: ...", and "mother: ..." can be omitted